runtime/jcl/CMakeLists.txt

```diff
@@ -115,6 +115,7 @@ target_sources(jclse
 		${CMAKE_CURRENT_SOURCE_DIR}/common/com_ibm_jvm_Stats.c
 		${CMAKE_CURRENT_SOURCE_DIR}/common/com_ibm_oti_vm_VM.c
 		${CMAKE_CURRENT_SOURCE_DIR}/common/compiler.c
+		${CMAKE_CURRENT_SOURCE_DIR}/common/criu.cpp
```

Comment:
This should be conditional like it is in UMA builds.

Reply:
Addressed via https://github.com/eclipse-openj9/openj9/pull/14947